[TM-515] update README before archiving

pasqu4le opened the pull request:
## Description

This repository will be archived, because of that we add a warning to the README.

## Related issue(s)

https://issues.serokell.io/issue/TM-515

## :white_check_mark: Checklist for your Pull Request

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

rvem commented:
We can archive this project without removing scheduled jobs from the `pipeline.yml`. Scheduled runs should be disabled in the [pipeline settings](https://buildkite.com/serokell/tezos-globacap/settings/schedules)

rvem reviewed (approved):
LGTM, have only comment

pasqu4le committed:
[TM-515] update README before archiving

Problem: this repository will be archived, because of that we want
to add a warning to the README.

Solution: update README.
